modules/nw-sriov-dpdk-example-mellanox.adoc

```diff
@@ -6,47 +6,48 @@
 [id="example-vf-use-in-dpdk-mode-mellanox_{context}"]
 = Using a virtual function in DPDK mode with a Mellanox NIC
 
+You can create a network node policy and create a Data Plane Development Kit (DPDK) pod using a virtual function in DPDK mode with a Mellanox NIC.
 .Prerequisites
 
-* Install the OpenShift CLI (`oc`).
-* Install the SR-IOV Network Operator.
-* Log in as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the Single Root I/O Virtualization (SR-IOV) Network Operator.
+* You have logged in as a user with `cluster-admin` privileges.
 
 .Procedure
 
-. Create the following `SriovNetworkNodePolicy` object, and then save the YAML in the `mlx-dpdk-node-policy.yaml` file.
+. Save the following `SriovNetworkNodePolicy` YAML configuration to an `mlx-dpdk-node-policy.yaml` file:
 +
 [source,yaml]
 ----
 apiVersion: sriovnetwork.openshift.io/v1
```

Comment:
I think this file does need relative indentation. What I meant by my earlier comment was that the top level indentation should start at the left of the page. Previously, some of the YAML examples had the top level indented to the right a few spaces. All of the lines should be indented according to the sections they are in, but the left-most level must have no spaces before it. Please let me know if you would like to discuss what I mean by this.